test/spec/EnrollU2F_spec.js

```diff
@@ -10,7 +10,7 @@ define([
   'sandbox',
   'LoginRouter',
   'helpers/xhr/MFA_ENROLL_allFactors',
-  'helpers/xhr/MFA_ENROLL_ACTIVATE_U2F'
+  'helpers/xhr/MFA_ENROLL_ACTIVATE_u2f'
```

Comment:
Case sensitivity in mac vs linux and it not actually failing the test suite when files can't be loaded.